vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -154,12 +158,14 @@ function Install-ContainerD {
         Expand-Archive -path $containerdTmpDest -DestinationPath $installDir -Force
     } else {
         tar -xzf $containerdTmpDest --strip=1 -C $installDir
+        mv -Force $installDir\bin\* $installDir
```

Comment:
mv => Move-Item? To keep the same format.

Reply:
I reverted the code the use mv instead.